dist/Makefile.tmpl

```diff
@@ -47,7 +47,7 @@ OBJS 		+= $(patsubst %.S,%.o,$(wildcard *-$(MARCH)$(VARIANT).S))
 
 include Makefile.basic
 
-CFLAGS		+= -Wno-parentheses -Wno-deprecated-declarations -Wno-type-limits -Wno-error=cpp -Wno-cpp -g -std=gnu11 -O3
+CFLAGS		+= -Wno-parentheses -Wno-deprecated-declarations -Wno-\#warnings -Wno-error=cpp -Wno-cpp -g -std=gnu11 -O3
```

Comment:
where does this change come from?

Reply:
This has always been like this. It just revers a change by Karthik on here https://github.com/cryspen/hacl-star/commit/0d14af79c4bad94b2de2a22ae90a52297f4ef181
Also, I'd always argue no one should use this anyway 🤷🏻